console.log("Hello")
console.log("Hello")
console.log("Hello")
//Jis code per doubt hota hai!
// let a=5;
try{
    console.log(a)
}
//Catch is important. when try statement gives error catch runs.
catch(error){
    console.log("Variable a don't found")
}
console.log("hello2")
console.log("hello2")
console.log("hello2")
